app/models/data.py

```diff
@@ -7,25 +7,91 @@
 from datetime import datetime
 from app import db
 
+from sqlalchemy.ext.declarative import declared_attr
+from sqlalchemy.inspection import inspect
+from sqlalchemy.orm import relationship
 
-class Batch(db.Model):
+class DataMixin(object):
+
+    @declared_attr
+    def __tablename__(cls):
+        return cls.__name__.lower()
+
+    def to_dict(self):
+        d = {}
```

Comment:
Is `return {col.name: str(getattr(self, col.name)) for col in self.__table_.columns}` more readable? ...no, right?